packages/in_app_purchase/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.0.0
+
+* **BREAKING CHANGE** Updates the dependency on `in_app_purchase_storekit` to `0.3.0+1` which ensures the `restorePurchases` method emits an empty list of purchases. 
```

Comment:
We should probably explicitly clarify that 2.0.0 did not do this as it said, so that people don't think they don't need to change anything. Maybe:

```
* **BREAKING CHANGE** Updates `restorePurchases` to emit an empty list of purchases on StoreKit when there are no purchases to restore (same as Android).
  * This change was listed in the CHANGELOG for 2.0.0, but the change was accidentally not included in 2.0.0.
```

(We should probably also strikethrough the line in the 2.0.0 notes to make them retroactively less misleading, without actually removing the line.)